static/panes/tool.js

```diff
@@ -36,6 +36,7 @@ var monacoConfig = require('../monaco-config');
 var ceoptions = require('../options').options;
 var utils = require('../utils');
 var PaneRenaming = require('../widgets/pane-renaming').PaneRenaming;
+var {saveAs} = require('file-saver');
```

Comment:
Tiny nit, can't use destructures in es5 code
```suggestion
var saveAs = require('./file-saver').saveAs;
```